docs/contributing.rst

```diff
@@ -4,68 +4,69 @@ Contributing
 
 Help is always appreciated!
 
-To get started, you can try :ref:`building-from-source` in order to familiarize
+To get started, you can try :ref:`building-from-source` to familiarise
 yourself with the components of Solidity and the build process. Also, it may be
 useful to become well-versed at writing smart-contracts in Solidity.
 
 In particular, we need help in the following areas:
 
-* Improving the documentation
+* Improving the documentation.
 * Responding to questions from other users on `StackExchange
   <https://ethereum.stackexchange.com>`_ and the `Solidity Gitter
-  <https://gitter.im/ethereum/solidity>`_
+  <https://gitter.im/ethereum/solidity>`_ channel.
 * Fixing and responding to `Solidity's GitHub issues
   <https://github.com/ethereum/solidity/issues>`_, especially those tagged as
   `up-for-grabs <https://github.com/ethereum/solidity/issues?q=is%3Aopen+is%3Aissue+label%3Aup-for-grabs>`_ which are
   meant as introductory issues for external contributors.
 
-Please note that this project is released with a `Contributor Code of Conduct <https://raw.githubusercontent.com/ethereum/solidity/develop/CODE_OF_CONDUCT.md>`_. By participating in this project - in the issues, pull requests, or Gitter channels - you agree to abide by its terms.
+This project is released with a `Contributor Code of Conduct <https://raw.githubusercontent.com/ethereum/solidity/develop/CODE_OF_CONDUCT.md>`_. By participating in this project - in the issues, pull requests, or Gitter channels - you agree to abide by its terms.
 
 How to Report Issues
 ====================
 
-To report an issue, please use the
+To report an issue, use the
```

Comment:
This document addresses contributors. They know where and how to file issues. If not, then this is not the right place to contribute.

Still, if you keep this, you should delete the instructions, as they are given in the issue-template.

"Use the Issue-Tracker (link) and follow the given instructions" should be enough.